pages/Category.vue

```diff
@@ -616,7 +616,14 @@ export default {
   padding: var(--spacer-base) var(--spacer-base) var(--spacer-base) var(--spacer-sm);
 }
 .navbar {
-  position: relative;
+  // Made it visible while scrolling.
```

Comment:
This is going to be applied to mobile and desktop. I think we should only have this active on mobile using a media query.

Reply:
Ok I will update this